csrc/pf_save.c

```diff
@@ -218,13 +218,12 @@ void WriteCellBigEndian( uint8_t *addr, ucell_t data )
 {
     /* Write should be in order of increasing address
      * to optimize for burst writes to DRAM. */
-    if( sizeof(ucell_t) == 8 )
-    {
+	#if (LONG_MAX > 2147483647)  /* LONG_MAX from limits.h: values > 2^31-1 indicates 64bit ints */
```

Comment:
Thanks for the idea. We start conditional compilation in column 0.
And we can use PF_SIZEOF_CELL, which is used in multiple places for similar purposes.
So it should be safe.

```suggestion
#if (PF_SIZEOF_CELL == 8)
```